### DISCUSSION TITLE
pyqtgraph not working with Pyside 6.4

### DISCUSSION THREAD

**Discussion by chstolle** [pyqtgraph/pyqtgraph#2631] [General]
Hi, I'm getting some strange errors after updating to PySide6 > 6.4.0 when trying to plot something:

```
NotImplementedError: pure virtual method 'QGraphicsObject.paint' not implemented.Traceback (most recent call last):
  File ".../lib/python3.10/site-packages/pyqtgraph/widgets/GraphicsView.py", line 137, in paintEvent
    return super().paintEvent(ev)
NotImplementedError: pure virtual method 'QGraphicsObject.paint' not implemented.
```

I'm using these packages:

```
pyqtgraph                 0.13.1 
pyside6                   6.4.2 
qtpy                      2.3.0 
```

The error seems to be occurring since PySide version 6.4.0. Version 6.3.1 is working fine. Does anybody else have this issue?

In the Qt Bindings Test Matrix there seems to be no official support for pyside6 yet. Does anybody know when it might be supported?

Thanks!

**Comment by pijyoi** [comment]
The bindings matrix documentation is outdated. For Qt6, 6.2 and the latest available version on pypi are tested. This currently means 6.4.2.

**Reply by j9ac9k** [reply]
@pijyoi didn't you add a PySide 6.4.x fix after 0.13.1 was released?

**Reply by pijyoi** [reply]
I can't remember. But I did also try running the SimplePlot on 0.13.1 with 6.4.2 with no issue.

**Reply by j9ac9k** [reply]
I'll be doing a 0.13.2 release soon enough.... as soon as I'm not falling asleep at 8 pm some evening...
